pkgs/development/compilers/glslang/default.nix

```diff
@@ -20,22 +49,22 @@ stdenv.mkDerivation rec {
 
   # These get set at all-packages, keep onto them for child drvs
   passthru = {
-    inherit spirv-tools spirv-headers;
+    inherit localSpirv-tools localSpirv-headers;
```

Comment:
Small nitpick - this is a change in the "API" of it - use `spirv-tools = localSpirv-tools`, etc.